qwed_infra/parsers/terraform_parser.py

```
@@ -1,32 +1,60 @@
         # 1. Read and merge generic HCL structure
-        for tf_file in path.glob("*.tf"):
+        for tf_file in sorted(path.glob("*.tf")):
             with open(tf_file, 'r') as f:
                 try:
                     data = hcl2.load(f)
-                    # Merge logic (simplified: appending lists)
                     for key, val in data.items():
                         if key not in combined_hcl:
                             combined_hcl[key] = []
                         combined_hcl[key].extend(val)
                 except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)